src/files-and-videos/files-page/FilesPage.test.jsx

```diff
@@ -205,7 +205,7 @@ describe('FilesAndUploads', () => {
           await mockStore(RequestStatus.SUCCESSFUL);
           axiosMock.onGet(`${getAssetsUrl(courseId)}?display_name=download.png&page_size=1`).reply(200, { assets: [] });
           axiosMock.onPost(getAssetsUrl(courseId)).reply(200, generateNewAssetApiResponse());
-          const addFilesButton = screen.getByLabelText('file-input');
+          const addFilesButton = screen.getByLabelText(messages.fileInputAriaLabel.defaultMessage);
```

Comment:
```suggestion
          const addFilesButton = screen.getByLabelText('Upload a file');
```
Nit: This may seem to go against the usual "DRY" developer maxim, but I generally prefer that strings are hard-coded in test files, because it makes it much more clear when things get changed accidentally. When the test references the code, it's easier to introduce a regression without noticing. In this case it's not a big deal though.

Reply:
I understand your point but lets keep it like this this time since it is not a major thing.